src/scene/scene.js

```diff
@@ -137,6 +138,14 @@ class Scene extends EventHandler {
      */
     lightmapFilterEnabled = false;
 
+    /**
+     * Enables HDR lightmaps. This can result in smoother lightmaps especially when many samples
+     * are used.
```

Comment:
```suggestion
     * Enables HDR lightmaps. This can result in smoother lightmaps especially when many samples
     * are used. Defaults to false.
```